libs/cart/driver/magento/src/errors/transform.ts

```diff
@@ -9,23 +13,29 @@ import {
 } from './map';
 
 
-function transformMagentoCartGraphQlError(error: ApolloError): Error {
+function transformMagentoCartGraphQlError(error: ApolloError, requestPayload?: any): Error {
```

Comment:
No explicit any.

Reply:
removed

Reply:
Oh, not that, my bad. What I meant was, this needs to be strictly typed. We can't use `any` we should either set it to `unknown` or narrow the type to something more usable than any.

Reply:
I think the best we can do is `unknown` unless we set it to a union of all the driver request types.

Reply:
I just typed it as `unknown`